hr_planning_resources/models/helpdesk_ticket.py

```diff
@@ -0,0 +1,20 @@
+from odoo import api, models
+
+
+class HelpdeskTicket(models.Model):
+    _name = "helpdesk.ticket"
+    _inherit = ["helpdesk.ticket", "hr.task.mixin"]
+
+    def _compute_hr_task_count(self):
+        for record in self:
+            ticket_count = self.env["hr.task"].search_count(
```

Comment:
Please use `read_group` (before the `for`) to do only a single query :pray:

Reply:
Also, consider moving this to a separate glue module with the Helpdesk app.
You don't want to force that app on the users.